cmd/kev/cmd/dev.go

```diff
@@ -127,7 +127,7 @@ func init() {
 	rootCmd.AddCommand(devCmd)
 }
 
-func runDevCmd(cmd *cobra.Command, args []string) error {
+func runDevCmd(cmd *cobra.Command, _ []string) error {
 	skaffold, err := cmd.Flags().GetBool("skaffold")
```

Comment:
It's not the scope of this PR necessarily but I think we could fix it as we touch commands anyway... either handle err for each parameter getter, or just skip that altogether, and err check below in line 139? This applies to this and other commands where we don't handle errors in place.